### DISCUSSION TITLE
ScriptReference/AssetDatabase.GetSubFolders

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#517] [Notes]
# ScriptReference/AssetDatabase.GetSubFolders

https://docs.unity3d.com/ScriptReference/AssetDatabase.GetSubFolders.html

**Comment by joethephish** [comment]
Each subfolder is also relative to the project. So, for example:

```
AssetDatabase.GetSubFolders("Assets/My Folder");      // [ "Assets/My Folder/Some subfolder", "Assets/My Folder/Another" ]
```
